requirements/base.txt

```diff
@@ -1,7 +1,7 @@
 # the default package dependencies
 
 numpy>=1.16.4
-torch>=1.3
+torch>=1.3, <1.6  # TODO: temporary freze fro Horowon incomaptibility with 1.6
```

Comment:
typo: `temporary freeze for Horovod incomaptibility with 1.6`

Reply:
mind use suggestions? ;]